egs/vcc2020/voc1/local/preprocess.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Nagoya University (Wen-Chin Huang)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+"""Perform preprocessing and raw feature extraction.
+   This is slightly different from the original version,
+   because we want to train 16kHz-24kHz PWG, 
+   so we skip sampling rate check.
+"""
```

Comment:
If the difference only the check, I can make `--allow-different-sampling-rate` option in PWG.

Reply:
Again, could you explain the difference from the original script?